src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,113 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] arrayList;
+    private int size;
+
+    public ArrayList() {
+        arrayList = (T[]) new Object[DEFAULT_CAPACITY];
+        size = 0;
+    }
 
     @Override
     public void add(T value) {
-
+        reviewSize();
+        arrayList[size++] = value;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index < size && index >= 0) {
+            reviewSize();
+            addingShift(index);
+            arrayList[index] = value;
+        } else {
```

Comment:
this else block is redundant, make it the way you did in get() method

Reply:
done